backend/Models/Database/Sightings.cs

```diff
@@ -0,0 +1,19 @@
+using System; 
+
+namespace WhaleSpotting.Models.Database
+{
+    public class Sightings
```

Comment:
Name in the singular (remember to rename the file)
```suggestion
    public class Sighting
```